tests/test_cli.py

```diff
@@ -672,7 +672,7 @@ def test_ask(
 
         assert output["trial"]["number"] == 0
         assert len(output["trial"]["params"]) == 2
-        assert 0 <= output["trial"]["params"]["x"] < 1
+        assert 0 <= output["trial"]["params"]["x"] <= 1
```

Comment:
Notes: `high` is included from the range. https://github.com/optuna/optuna/blob/3418f002231a88b2e0b189ed8f97bbd98d66d9ec/optuna/distributions.py#L105-L106
https://github.com/optuna/optuna/blob/3418f002231a88b2e0b189ed8f97bbd98d66d9ec/optuna/distributions.py#L131